### PR TITLE
Additional test models for keyword PINCH

### PR DESCRIPTION
**Pull request by tskille**
A total of 23 models. These are very small and runs in less that 0.1 s (each model)

A description of each model can be found in header of data file. The markup file has also been updated.

### CODE REVIEW

File: pinch/T2A_NOPINCH.DATA [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+-- Test model for keyword PINCH and grids where there is a gaps between layer K and K+1.
+-- The model includes a Grid with dimension 1x1x3 where here is a gap of 1.0 m between 
+-- layer 2 and 3.
+
+-- The model includes 1 well completed in cell 1,1,1. Summary vectors 
+-- BPR (block pressure) will reveal whether there is communication or not 
+-- from layer 2 to 3. 
+
+-- Since PINCH keyword is not used in this deck, there should be communication
+-- across layer 2 and 3. Hence Bottom layer should not be depleted.
```
Comment (bska) [inline pinch/T2A_NOPINCH.DATA:17]: Either I'm reading it incorrectly or this statement is self-contradictory.  Is there a "no" missing in "there should be communication" here, or should the second sentence read "... should be depleted"?  By the way, the `PINCH.md` file says there should be a non-zero `TRANZ` value for this case (T2A_NOPINCH).

**Review by bska — COMMENTED** [review]
This is a massive contribution&mdash;thank you so much!  I only noticed one minor inconsistency in the `T2A_NOPINCH` model description.  There are some misprints, but we can address those later if you don't wish to do it now.

File: pinch/T2A_NOPINCH.DATA (hunk 1 above)
Comment (tskille) [reply pinch/T2A_NOPINCH.DATA]: Thanks @bska , the inconsistency in the header of T2A_NOPINCH has been updated.  There should be communion and bottom layer should be depleted.

**Review by bska — APPROVED** [review]
Sorry about the delay.  This looks good to me now and I'll merge into master.

**Pull request merged**